packages/next-drupal/src/client.ts

```diff
@@ -1359,14 +1359,23 @@ export class DrupalClient {
 
     const basic = Buffer.from(`${clientId}:${clientSecret}`).toString("base64")
 
-    const response = await fetch(url.toString(), {
+    const options = {
       method: "POST",
       headers: {
         Authorization: `Basic ${basic}`,
         "Content-Type": "application/x-www-form-urlencoded",
       },
       body: `grant_type=client_credentials`,
-    })
+    };
+
+    // Initializing empty response object.
+    let response = new Response();
```

Comment:
I think this all could be done on a single line using ternary operator. Something like:

 ```
const response = this.fetcher ? await this.fetcher(url.toString(), options) : await fetch(url.toString(), options);
```

Reply:
Yes! we can use the ternary but I used the pattern same as https://github.com/chapter-three/next-drupal/blob/main/packages/next-drupal/src/client.ts#L306 with the debug message